### PR TITLE
Highlight current 186

### LINKED ISSUE #186: Discuss: transcript sync button? transcript search box?
The transcript scrolling/synching looks fantastic @mccalluc! Very well done!

For the sake of making more work for us, I wanted to discuss two features that the old version of OV had for transcripts. These may be of limited use, and we may not want to implement them, but thought I'd bring them up.

There was a button that would re-sync the transcript to the current location of the media. This would be useful if the user has scrolled through the transcript independent of the media, thus deactivating the sync.

Also, the old version had a mini-search box for searching just the transcript. Not sure how useful this would be. There are a lot of jquery options out there for doing this, plus highlighting found terms, etc.

Style-wise, I was playing around and here's something I came up with. A menu for the transcript pages that tries not to get in the way...
image

... and it's partially transparent...
image

I realize this is a little bit of self-imposed scope creep, but it _is_ a feature that used to be there that isn't any longer.

Thoughts on the feature? Thoughts on the style idea?

### LINKED ISSUE #252: Start player at given timecode
In the Advocates collection, the article really wants to be able to play clips from a given timecode... Casie is looking into whether the article is a work-for-hire, and whether we can edit it to make this less necessary.

### PR DESCRIPTION
**Pull request by mccalluc**
@afred? There are really several new features in this PR:
- mouseover on the player will also scroll the transcript.
- roundrect around the current line of the interview. (No strong preferences on style, though background shading seemed like too much to me?)
- url hash to go to middle of file. (No strong preference on the format of the hash.)
- and, if you click on a line of the transcript, it adds a hash to the URL: seems like an unobtrusive way to expose this to the user?

Fix #252, Fix #186 (taken together with the other PR)

### CODE REVIEW

File: app/assets/javascripts/player.js [hunk 1]
```diff
@@ -9,12 +9,12 @@ $(function(){
     // TODO: offsets need to be recalculated when width changes.
```
Comment (afred) [inline app/assets/javascripts/player.js]: @mccalluc can remove this comment, yes?

Comment (mccalluc) [inline app/assets/javascripts/player.js]: Good catch: the new way fixes that problem, though I wasn't thinking out it.

**Comment by mccalluc** [comment]
@afred: if no other issues, ok to merge?

**Comment by afred** [comment]
@mccalluc checking.. i found a small jumping behavior in Chorme.. but if that's all, then I'll file a new ticket for it. Stand by.

**Pull request merged**